homeassistant/helpers/dispatcher.py

```diff
@@ -151,11 +151,11 @@ def _format_err[*_Ts](
     *args: Any,
 ) -> str:
     """Format error message."""
-    return "Exception in {} when dispatching '{}': {}".format(
+
+    return (
         # Functions wrapped in partial do not have a __name__
-        getattr(target, "__name__", None) or str(target),
-        signal,
-        args,
+        f"Exception in {getattr(target, '__name__', None) or target} "
+        f"when dispatching '{signal}': {args}"
```

Comment:
Minor nitpick: Given that HA requires Python v3.12, you could take advantage of the fact that double quotes are allowed inside f-strings, if just for the sake of using consistent quotation marks.

No strong opinion about this, still mentioning it in case you didn’t notice.

Reply:
Done